lib/repo/git_repository.rb

```diff
@@ -30,24 +30,24 @@ class GitRepository < Repository::AbstractRepository
     # created using GitRespository.create(), it it is not yet existent
     def initialize(connect_string)
       # Check if configuration is in order
-      if Repository.conf[:IS_REPOSITORY_ADMIN].nil?
+      if MarkusConfigurator.markus_config_repository_admin?.nil?
         raise ConfigurationError.new(
-                "Required config 'IS_REPOSITORY_ADMIN' not set")
+                "Required config 'MarkusConfigurator.markus_config_repository_admin?' not set")
       end
-      if Repository.conf[:REPOSITORY_STORAGE].nil?
+      if MarkusConfigurator.markus_config_repository_storage.nil?
         raise ConfigurationError.new(
-                "Required config 'REPOSITORY_STORAGE' not set")
+                "Required config 'MarkusConfigurator.markus_config_repository_storage' not set")
```

Comment:
Line is too long. [96/80]<br>Indent the first parameter one step more than the start of the previous line.